tardis/model/density.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+
+from tardis.util import quantity_linspace
+
+
+class HomologousDensity(object):
+    def __init__(self, density_0, time_0):
+        self.density_0 = density_0
+        self.time_0 = time_0
+
+    def after_time(self, time_explosion):
```

Comment:
`calculate_density_evolution`? @chvogl If no other alternative in the next hour just use that and we are done!!